nodejs/packagescanner.go

```diff
@@ -113,7 +115,7 @@ func (s *Scanner) Scan(ctx context.Context, layer *claircore.Layer) ([]*claircor
 		if sv, err := semver.NewVersion(pkgJSON.Version); err == nil {
 			pkg.NormalizedVersion = claircore.FromSemver(sv)
 		} else {
-			zlog.Info(ctx).
+			zlog.Warn(ctx).
```

Comment:
@hdonnay would you prefer I revert this?

Reply:
Yeah. As a rule of thumb, we shouldn't warn if the request will continue. At least that's what I wrote down previously: https://github.com/quay/claircore/blob/main/docs/contributor/logging.md#levels

Reply:
reverted. should be ready for (hopefully) one last pass